README.md

```diff
@@ -1,129 +1,108 @@
 # NED
 
-A tool for setting up nitro on AWS
+NED is short for Newman's Environment for Development and is a nod to the
+great computer programmer Dennis Nedry.
+
+This project is a tool for helping with setting up development and test
+environments for Newman specifically on AWS and maybe other cloud providers
+in the future.
 
-## Setting up for development
+## Getting started
 
-Following [this
-guide](https://ealizadeh.com/blog/guide-to-python-env-pkg-dependency-using-conda-poetry),
-this project uses Conda for environment management, pip as the package
-installer, and Poetry as the dependency manager.
+This tool is intended to help you manage your own development environment(s)
+and not be used for managing standing infrastructure.  As such, you will
+probably want to fork this repo, make it your own, and maybe contribute some
+tools back if they are more generally useful.
 
-Install [Miniconda3](https://docs.conda.io/en/latest/miniconda.html#linux-installers).
+To get started, fork and clone the repo.
 
-    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    bash Miniconda3-latest-Linux-x86_64.sh
+Next, you will want to crate a configuration for the infrastructure that
+you would like to run.  To do so, create a file `terraform/main.tf` that
+describes your development infrastructure.  For example,
+my `main.tf` looks like this:
 
-Install [Mamba](https://github.com/mamba-org/mamba) to speed up environment building
+```hcl
+module "nitro_dev" {
+  source = "./modules/nitro"
+  instance_name = "dlm-terraform-test"
+  key_pair_name = "dlm-dev"
+}
```

Comment:
From our conversation about how to include some standard terraform stuff in the main.tf file, maybe we could update this readme chunk to look like the following:
```hcl
terraform {
  required_version = ">= 1.7.0"
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "5.58.0"
    }
  }
}

provider "aws" {
  # Note: Not all nitro-enabled instance types or amis are available in all regions. For now, use `us-east-1` to ensure compatibility with the module.
  region = "us-east-1"
}

module "nitro_dev" {
  source = "./modules/nitro"
  # Change the following to your own values
  instance_name = "example-instance-name"
  key_pair_name = "example-key-pair-name"
}

output "nitro_dev_instance_ip" {
  description = "The public IP address of the instance"
  value       = module.nitro_dev.instance_public_ip
}
```